Sources/TabBar/View/TabBar.swift

```diff
@@ -96,30 +96,32 @@ public struct TabBar<TabItem: Tabbable, Content: View>: View {
             .frame(maxWidth: .infinity)
         }
     }
+
     
-    public var body: some View {
-        ZStack {
-            self.content
-                .frame(maxWidth: .infinity, maxHeight: .infinity)
-                .environmentObject(self.selectedItem)
-            
-            GeometryReader { geometry in
-                VStack {
-                    Spacer()
-                    
-                    self.tabBarStyle.tabBar(with: geometry) {
-                        .init(self.tabItems)
-                    }
-                }
-                .edgesIgnoringSafeArea(.bottom)
-                .visibility(self.visibility)
-            }
-        }
-        .onPreferenceChange(TabBarPreferenceKey.self) { value in
-            self.items = value
-        }
-    }
-    
+	public var body: some View {
+		GeometryReader { geometry in
+			ZStack {
+				self.content
+					.environmentObject(self.selectedItem)
+					.padding(.bottom, 50)
```

Comment:
Could you please clarify why we need this padding here, and what does value of 50 points mean?

Reply:
Child view of TabBar collides with TabBar without this. 50 is the DefaultTabBarStyle tabbar height. Bottom 50px of view is not visible without this padding.